meinberlin/apps/bplan/api.py

```diff
@@ -31,3 +33,7 @@ def get_serializer_context(self):
             }
         )
         return context
+
+    def list(self, request, *args, **kwargs):
```

Comment:
the bplan api only allows `create` or `update` , not list - so I think we don't actually need caching here (unless I am missing something, sorry)

Reply:
good catch! I will remove it.